src/main/resources/swagger/api-docs.yaml

```diff
@@ -501,6 +501,11 @@ definitions:
           The archive must be in tar.gz format, must not include a parent directory, and
           must have an entry point named 'main'. For more information on notebook extensions,
           see http://jupyter-notebook.readthedocs.io/en/latest/extending/frontend_extensions.html.
+      jupyterUserScriptUri:
+        type: string
+        description: |
+          Optional bucket URI to an archive containing a bash script the user wishes to run inside their jupyter docker.
+          The archive must be in .sh format and must not include a parent directory.
```

Comment:
Is it an archive or a bash script? (It should just be a batch script; no need to zip it up.)